src/test/java/org/sagebionetworks/bridge/DefaultStudyBootstrapperTest.java

```diff
@@ -123,7 +123,7 @@ public void createsDefaultStudyWhenMissing() {
         assertEquals(SHARED_SUBPOP, subpopCaptor.getAllValues().get(2));
         
         StudyParticipant admin = participantCaptor.getAllValues().get(0);
-        assertEquals(admin.getRoles(), ImmutableSet.of(ADMIN, RESEARCHER));
+        assertEquals(admin.getRoles(), ImmutableSet.of(SUPERADMIN, RESEARCHER));
```

Comment:
Minor: SUPERADMIN can do everything that RESEARCHER can do, so you only need to add SUPERADMIN.